internal/cmd/tree.go

```diff
@@ -72,7 +72,13 @@ func (t *Tree) RenderPackage(ctx context.Context, srcPath string, opts ...Render
 		return "", fmt.Errorf("loading package contents from folder: %w", err)
 	}
 
-	pkg, err := packagecontent.PackageFromFiles(ctx, t.scheme, files, cfg.Component)
+	// TODO: show all components in the tree
```

Comment:
Instead of listing EVERYTHING with tree, wouldn't it make more sense if we add a `--component` flag to the tree command to only render the component in question?
At least that would be more what I would expect to happen.

Reply:
will do in a following PR 👍